assets/app/index.html

```diff
@@ -16,12 +16,59 @@
     <!-- build:css(.tmp) styles/main.css -->
     <link rel="stylesheet" type="text/css" href="styles/main.css">
     <!-- endbuild -->
-     <style type="text/css">
+    <style type="text/css">
 
 
     </style>
+    <!-- build:js(.) scripts/oldieshim.js -->
```

Comment:
can you put the script location back where it was for now? I'd prefer to make that change in it's own commit, rather than along-side a big change... makes it easier to track down resulting problems